icubam/messaging/scheduler_test.py

```diff
@@ -139,7 +139,7 @@ def test_schedule_all(self):
     self.assertGreater(len(self.scheduler.timeouts), len(names) + 1)
 
   @mock.patch('time.time', mock.MagicMock(return_value=fake_now))
-  def test_schedule_all(self):
+  def test_schedule_all_2(self):
```

Comment:
this one is probably not useful

Reply:
This one is actual critical: there are two test methods `test_schedule_all` now in this file and only one is being run by pytest. Granted this name could be improved, any suggestions?